src/ServiceControl/CustomChecks/CustomChecksComponent.cs

```diff
@@ -34,7 +34,7 @@ public override void Configure(Settings settings, IHostBuilder hostBuilder)
                         serviceCollection.AddSingleton<ICustomChecksDataStore, RavenDbCustomCheckDataStore>();
                         break;
                     case DataStoreType.SqlDb:
-                        serviceCollection.AddSingleton<ICustomChecksDataStore>(sp => new SqlDbCustomCheckDataStore(settings.SqlStorageConnectionString, sp.GetService<IDomainEvents>()));
+                        serviceCollection.AddSingleton<ICustomChecksDataStore>(sp => new SqlDbCustomCheckDataStore(settings.SqlStorageConnectionString));
```

Comment:
I saw that the `DomainEvents` wasn't required and have also removed it from this code in my [PR](https://github.com/Particular/ServiceControl/pull/2933) - perhaps we should try and merge these 2 together

Reply:
Rebasing with the branch solved this.